test/test_helper.rb

```diff
@@ -0,0 +1,14 @@
+require "rails"
+require "measured"
+require "measured-rails"
+require "minitest/autorun"
+require "mocha/setup"
+require "pry"
+
+require File.expand_path("../dummy/config/environment", __FILE__)
+
+ActiveSupport.test_order = :random
+
+class ActiveSupport::TestCase
+
+end
```

Comment:
This file loads the gem and the dummy app. No tests yet, but the test suite runs.

```
~/s/measured-rails(version-0.0.1)$ be rake test
/Users/kevin/.rbenv/versions/2.1.1/bin/ruby -I"lib:test:lib/**/*" -I"/Users/kevin/.rbenv/versions/2.1.1/lib/ruby/gems/2.1.0/gems/rake-10.4.2/lib" "/Users/kevin/.rbenv/versions/2.1.1/lib/ruby/gems/2.1.0/gems/rake-10.4.2/lib/rake/rake_test_loader.rb" "test/active_record_test.rb"
Run options: --seed 56261

# Running:

S

Finished in 0.002140s, 467.2897 runs/s, 0.0000 assertions/s.

1 runs, 0 assertions, 0 failures, 0 errors, 1 skips

You have skipped tests. Run with --verbose for details.
```